### PR TITLE
Add Dependent Destroy

### PR DESCRIPTION
**Pull request by michaelwehrley**
* Change associations so all likes and comments get deleted when and OTMS is deleted
* Update specs to test for deletion

[Finished: #104572752]
[#110256638]

### CODE REVIEW

File: spec/models/social_networking/on_the_mind_statement_spec.rb [hunk 1]
```diff
@@ -45,5 +46,41 @@ module SocialNetworking
 
       expect(otms.shared_description).to eq("I'm writing tests.")
     end
+
+    describe "when a statement exists" do
```
Comment (ericcf) [inline spec/models/social_networking/on_the_mind_statement_spec.rb]: My only question is: did you intentionally make all of the participant associations the same model, or did you want to have a separate participant create the comment?

Comment (michaelwehrley) [inline spec/models/social_networking/on_the_mind_statement_spec.rb]: That was a mistake - THANK YOU! :arg!

Comment (michaelwehrley) [inline spec/models/social_networking/on_the_mind_statement_spec.rb]: @ericcf test driven development would have caught that.  Okay, all set.  Thanks!

**Comment by ericcf** [comment]
:boom:

**Pull request merged**